doc/source/reference/series.rst

```diff
@@ -228,6 +226,9 @@ Reshaping, sorting
    Series.argmin
    Series.argmax
    Series.reorder_levels
+   Series.nlargest
+   Series.nsmallest
+   Series.nsorted
```

Comment:
I added nsorted and moved the sorting method to the "Reshaping, sorting" section